8.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-unused-vars */
+
+// Q 8: Find the thirteen adjacent digits in the 1000-digit number that have the greatest product. What is the value of this product?
+
+// question to mike: when i use this code, the answer gives 0, and the array from getArrays has NaN inside it. works with smaller numbers. 
+// is this to do with the code or the number being too large?
```

Comment:
It's likely the number is too large!   You have to find a way to spit out 13 digits in a sliding window kind of way. :)